### PR TITLE
Add blog article about Modtoberfest and Hacktoberfest, and page about contributing to Quilt as part of them.

### PR DESCRIPTION
**Pull request by Southpaw1496**
I will update the date on the blog post before the PR is merged, to make sure it is accurate.

### CODE REVIEW

File: src/pages/en/blog/2022-10-02-october-month-of-the-toberfests.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## How do I start?
+You can start by going to [https://hacktoberfest.com](https://hacktoberfest.com) and/or [https://modtoberfest.com/](https://modtoberfest.com/) and signing up. From there, all you need to do is find a participating project, see what you can contribute, and start coding!
+
+We have a full page with [guidelines for making contributions to Quilt specifically](https://quiltmc.org/toberfests), but a word of advice: maintainers of the projects you contribute to have to officially accept the pull requests that you open in order for it to contribute to your total, and most of them will only accept ones that are high-quality and show effort put in for its own sake, not tiny changes that are clearly made in pursuit of a t-shirt (or the opportunity to be able to hold a Pineapple in one's hands): your contributions should be made with love and genuinely benefit the project that you're submitting to. Make sure to discuss any changes you want to make with the project's maintainers before contributing, and make sure to follow the project's contribution guidelines from the start to minimize their workload in the review phase, and maximize your chances of getting accepted.
```
Comment (pluiedev) [inline src/pages/en/blog/2022-10-02-october-month-of-the-toberfests.md:25]: The link should be localized—currently I think the convention is to write direct links without relying on the default English redirect, so
```diff
- [guidelines for making contributions to Quilt specifically](https://quiltmc.org/toberfests)
+ [guidelines for making contributions to Quilt specifically](https://quiltmc.org/en/toberfests) 
```

File: src/pages/en/toberfests.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+### Setting up a GitHub account
+GitHub is a service that hosts git repositories (that is, folders containing files tracked by git), so that people can easily download and work on them together. We'll get more into how git works later, but the first step is to go to [https://github.com/signup](https://github.com/signup) and create an account. Once that is complete, click your avatar in the top-right, and click Settings. There are a lot of settings here that you can dig through if you want, but there's only a couple that need to be changed. Click "Email" in the sidebar and ensure that "Keep my email address private" and "Block command line pushes that expose my email" are both ticked. This is to ensure that you don't accidentally leak your email address, as all git commits must include an email address, but with these settings enabled, GitHub will by default give you a public address to associate your commits with, and stop you from accidentally uploading commits that use your personal email.
```
Comment (pluiedev) [inline src/pages/en/toberfests.mdx:13]: I think just `https://github.com/signup` would do the trick
```diff
- [https://github.com/signup](https://github.com/signup)
+ https://github.com/signup
```

File: src/pages/en/toberfests.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### Using github.dev to make your changes
+Installing and configuring git on your local machine can be a complex process, but if you're not making any complex changes (for example, if you're writing documentation or doing translation), it can be easier to use github.dev, which is a near-full featured version of the Visual Studio Code editor that runs in your browser. All the commits you make are sent directly to GitHub, so there's no need to run git locally. To use github.dev, all you have to do is visit your forked repository and press the `.` key on your keyboard, and you'll be taken to a VSCode instance running in the browser, with your repository opened and ready for editing. To keep following this guide, skip down to "Commits".
+
+### Installing git locally
+If you would prefer to work locally, there are several ways you can install git yourself, depending on your operating system:
+
+#### Windows
+You can download Git for Windows [here](https://git-scm.com/download/win)
```
Comment (pluiedev) [inline src/pages/en/toberfests.mdx:25]: This is bad for screen reader accessibility—a direct link would do anyway
```md
#### Windows
You can download Git for Windows at https://git-scm.com/download/win.
```

File: src/pages/en/toberfests.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+![Screenshot showing the Branch Selection interface of Visual Studio Code](/assets/img/writing/toberfest/toberfest-5.png)
+### Pull Requests
+The last thing you need to know about Git and GitHub is the concept of Pull Requests. Pull requests are GitHub's system for sending changes between branches, or, more importantly, from a fork of a repository back to the original one. To create a Pull Request, visit your fork on GitHub, click current branch in the top left, and select the branch with your changes to switch to it.
+
+![Screenshot highlighting the branch switching interface on github.com](/assets/img/writing/toberfest/toberfest-6.png)
+
+Then click "Contribute" and then click "Create Pull Request". More information about what you need to include in your Pull Requests can be found further down the page.
+
+And that is the basics of how to use git and GitHub. If you're interested in learning more about how git works, there is an excellent visual tutorial available at [https://learngitbranching.js.org](https://learngitbranching.js.org). If you get stuck, don't be afraid to ask for help on [Discord](https://discord.quiltmc.org/toolchain).
```
Comment (pluiedev) [inline src/pages/en/toberfests.mdx:68]: Again direct links should just work:
```diff
- [https://learngitbranching.js.org](https://learngitbranching.js.org)
+ https://learngitbranching.js.org
```
And that we should use the `site` variables for links to our Discord server and stuff like that
```diff
---
# ...
---
+ import site from "@data/Site";

<!-- snip -->

- to ask for help on [Discord](https://discord.quiltmc.org/toolchain).
+ to ask for help on <a href={site.discordCommunity}>Discord</a>.

File: src/pages/en/toberfests.mdx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- [Quilt Mappings](https://github.com/quiltmc/quilt-mappings)
+- [Quilt Standard Libraries](https://github.com/quiltmc/quilt-standard-libraries)
+
+In addition, the following Quilt projects are accepting **low or non-code** Pull Requests for Hacktoberfest:
+- Developer Wiki (documentation)
+- quiltmc.org (Translation)
+
+After you decide on a repository to contribute to, a great place to start is by looking at the Issues tab for something you think you could help with, unless you already have something in mind. If you find something that you think you can do, please leave a comment stating your intent to work on it, and please clarify any details you're unsure of before starting work. Likewise, if you're attempting something that isn't being tracked in an issue, **please** confirm that it is something that would be beneficial to the project before you start, either by making an issue on GitHub or chatting to the developers on [Discord](https://discord.quiltmc.org/toolchain); we'd hate for your work to go to waste! We will not accept contributions that are obviously low-effort or unhelpful, and PRs will be accepted at the sole discretion of the team responsible for the repository.
```
Comment (pluiedev) [inline src/pages/en/toberfests.mdx:87]: Should use the `discordToolchain` variable from `@data/Site`
```diff
- [Discord](https://discord.quiltmc.org/toolchain)
+ <a href={site.discordToolchain}>Discord</a>
```
Also I feel like we should clarify that we have two Discord servers? Currently that is not very clear

**Review by pluiedev — CHANGES_REQUESTED** [review]

File: src/pages/en/toberfests.mdx (hunk 5 above)
Comment (Southpaw1496) [reply src/pages/en/toberfests.mdx:87]: I'm only linking to Toolchain because it's the one focused on development, so I think it's OK.

**Comment by Southpaw1496** [comment]
Closing because it's authored under the wrong email and will cause problems

**Pull request closed without merging**